src/__tests__/conditional.spec.ts

```diff
@@ -127,7 +128,7 @@ describe('buildkite-graph', () => {
         });
       });
 
-      describe.only('isEffectOf', () => {
```

Comment:
@joscha was that a mistake?

Reply:
Yes, did we break something? Should really add a linting rule.

Reply:
at least all tests passed.. :p